src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -266,17 +294,19 @@ public enum RisingRivals implements LogicCardInfo {
           move "Incite", {
             text "Search your discard pile for up to 2 Supporter cards, show them to your opponent, and put them into your hand."
             energyCost ()
-            attackRequirement {}
+            attackRequirement {
+              assert my.discard.filterByType(SUPPORTER) : "You have no supporter cards in your discard pile"
+            }
             onAttack {
-              damage 0
+              my.discard.select(max:2, "Search you discard pile for up to 2 Supporter cards",cardTypeFilter(SUPPORTER)).showToOpponent("Incite: These cards will be returned to your opponent's hand").moveTo(my.hand)
```

Comment:
You need a minimum of 1 if the text is "up to". Once you declare the attack you must do the effect.

Reply:
I may be wrong, but I though select always has a minimum of 1 by default. Is there a case where not explicitly setting a minimum would allow for 0 cards to be selected?

Reply:
I think you might be right. I always remember in the back of my head that Deck searching has a minimum of 0, so I always try to double check when I see a select. I think this is fine though.